[32mINFO    [0m HTTP_CLIENT:event_hooks.py:30 Make POST request to http://localhost:8000/api/v1/users
[32mINFO    [0m HTTP_CLIENT:event_hooks.py:40 Got response 200 OK from http://localhost:8000/api/v1/users
[32mINFO    [0m HTTP_CLIENT:event_hooks.py:30 Make POST request to http://localhost:8000/api/v1/authentication/login
[32mINFO    [0m HTTP_CLIENT:event_hooks.py:40 Got response 200 OK from http://localhost:8000/api/v1/authentication/login
[32mINFO    [0m HTTP_CLIENT:event_hooks.py:30 Make POST request to http://localhost:8000/api/v1/files
[32mINFO    [0m HTTP_CLIENT:event_hooks.py:40 Got response 200 OK from http://localhost:8000/api/v1/files
[32mINFO    [0m HTTP_CLIENT:event_hooks.py:30 Make GET request to http://localhost:8000/api/v1/files/incorrect-file-id
[32mINFO    [0m HTTP_CLIENT:event_hooks.py:40 Got response 422 Unprocessable Entity from http://localhost:8000/api/v1/files/incorrect-file-id
[32mINFO    [0m BASE_ASSERTIONS:base.py:19 Check that response status code equals to '422'
[32mINFO    [0m FILES_ASSERTIONS:files.py:155 Check get file with incorrect file id
[32mINFO    [0m ERRORS_ASSERTIONS:errors.py:40 Check validation error response
[32mINFO    [0m BASE_ASSERTIONS:base.py:73 Check that length of details equals to 1
[32mINFO    [0m ERRORS_ASSERTIONS:errors.py:20 Check validation error
[32mINFO    [0m BASE_ASSERTIONS:base.py:38 Check that 'type' equals to uuid_parsing
[32mINFO    [0m BASE_ASSERTIONS:base.py:38 Check that 'input' equals to incorrect-file-id
[32mINFO    [0m BASE_ASSERTIONS:base.py:38 Check that 'context' equals to {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1'}
[32mINFO    [0m BASE_ASSERTIONS:base.py:38 Check that 'message' equals to Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `i` at 1
[32mINFO    [0m BASE_ASSERTIONS:base.py:38 Check that 'location' equals to ['path', 'file_id']
[32mINFO    [0m SCHEMA_ASSERTIONS:schema.py:21 Validating JSON schema